CMakeLists.txt

```diff
@@ -870,6 +874,11 @@ if (Seastar_SSTRING)
     PUBLIC SEASTAR_SSTRING)
 endif ()
 
+if (SEASTAR_HAVE_EXECINFO)
```

Comment:
i'd move the `include()` and `check_include_file_cxx ()` call closer to where `SEASTAR_HAVE_EXECINFO` is used.

Reply:
also, we tend to use `Seastar_HAVE_EXECINFO`  as variable name in CMake.

Reply:
okay

Reply:
> i'd move the `include()` and `check_include_file_cxx ()` call closer to where `SEASTAR_HAVE_EXECINFO` is used.

@a16bitsysop could you reconsider this suggestion?

Reply:
I moved after seastar target is declared, it can’t go before as otherwise target doesn’t exist